<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>999</width>
    <height>693</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>321</width>
      <height>521</height>
     </rect>
    </property>
   </widget>
   <widget class="QTreeWidget" name="treeWidget">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>20</y>
      <width>571</width>
      <height>251</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Key</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Value</string>
     </property>
    </column>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>280</y>
      <width>571</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,7,1,1">
     <item>
      <widget class="QLabel" name="label_sections">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="ComboBox_section"/>
     </item>
     <item>
      <widget class="QPushButton" name="Btn_add_section">
       <property name="minimumSize">
        <size>
         <width>1</width>
         <height>1</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="resource.qrc">
         <normaloff>:/rec/img/plus.png</normaloff>:/rec/img/plus.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Btn_delete_section">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="resource.qrc">
         <normaloff>:/rec/img/remove.png</normaloff>:/rec/img/remove.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>370</y>
      <width>571</width>
      <height>171</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QGroupBox" name="groupBox_add">
       <property name="title">
        <string>GroupBox</string>
       </property>
       <widget class="QLabel" name="label_type">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>55</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_key">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>60</y>
          <width>55</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_value">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>90</y>
          <width>55</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
       <widget class="QComboBox" name="ComboBoxType">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>30</y>
          <width>171</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_key">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>60</y>
          <width>113</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_value">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>90</y>
          <width>113</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_add_key">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>120</y>
          <width>81</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>czcz</string>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_del">
       <property name="title">
        <string>GroupBox</string>
       </property>
       <widget class="QLabel" name="label_k">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>30</y>
          <width>55</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
       <widget class="QComboBox" name="ComboBoxKey">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>30</y>
          <width>141</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="Btn_delete_s">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>90</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>999</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="openFile"/>
    <addaction name="separator"/>
    <addaction name="openDirectory"/>
    <addaction name="separator"/>
    <addaction name="addFile"/>
    <addaction name="separator"/>
    <addaction name="saveFile"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="openFile"/>
   <addaction name="separator"/>
   <addaction name="openDirectory"/>
   <addaction name="separator"/>
   <addaction name="addFile"/>
   <addaction name="separator"/>
   <addaction name="saveFile"/>
  </widget>
  <action name="openFile">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/rec/img/openFile.png</normaloff>:/rec/img/openFile.png</iconset>
   </property>
   <property name="text">
    <string>Открыть файл</string>
   </property>
   <property name="toolTip">
    <string>Открыть файл</string>
   </property>
  </action>
  <action name="openDirectory">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/rec/img/openDirectory.png</normaloff>:/rec/img/openDirectory.png</iconset>
   </property>
   <property name="text">
    <string>Окрыть директорию</string>
   </property>
  </action>
  <action name="addFile">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/rec/img/newFile.png</normaloff>:/rec/img/newFile.png</iconset>
   </property>
   <property name="text">
    <string>Создать файл</string>
   </property>
   <property name="toolTip">
    <string>Создать файл</string>
   </property>
  </action>
  <action name="saveFile">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/rec/img/saveFile.png</normaloff>:/rec/img/saveFile.png</iconset>
   </property>
   <property name="text">
    <string>Сохранить файл</string>
   </property>
   <property name="toolTip">
    <string>сохранить файл</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
